sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -559,6 +559,17 @@ static std::unique_ptr<RawResponse> CreateHTTPResponse(
   end = std::find(start, last, '\r');
   auto reasonPhrase = std::string(start, end); // remove \r
 
+  // We currently only support HTTP/1.1, both in the request and the response.
+  if (majorVersion != 1 || minorVersion != 1)
+  {
+    Log::Write(
+        Logger::Level::Verbose,
+        LogMsgPrefix + "Unsupported HTTP version in the response. Only HTTP/1.1 is supported.");
```

Comment:
I don't know if logging here is necessary. Just did it out of caution. Thoughts on keeping/removing it?